src/lib/transaction_logic/test/zkapp_cmd_builder.ml

```diff
@@ -31,27 +31,35 @@ let get_nonce_exn (pk : Public_key.Compressed.t) :
   in
   nonce
 
-let update_body ?(update = Account_update.Update.noop) ~account amount =
+let update_body ?preconditions ?(update = Account_update.Update.noop) ~account
+    amount =
   let open Monad_lib.State.Let_syntax in
   let open Account_update in
-  let%map nonce = get_nonce_exn account in
+  let account_id = Test_account.account_id account in
+  let%map default =
+    let%map nonce = get_nonce_exn account.pk in
+    Account_update.Preconditions.
+      { network = Zkapp_precondition.Protocol_state.accept
+      ; account = Account_precondition.Nonce nonce
+      ; valid_while = Ignore
+      }
+  in
+  let update_preconditions = Option.value ~default preconditions in
```

Comment:
```suggestion
  let preconditions = Option.value ~default preconditions in
```

Reply:
Again, the name `preconditions` is shadowed by a function in `Account_update.Body`.